components/grid/grid.css

```diff
@@ -0,0 +1,33 @@
+.grid-layout{
+  display: grid;
+  grid-template-rows: 1fr;
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
```

Comment:
suggestion - if u are not giving gaps, remove these properties

Reply:
done made the changes thank you